spec/rspec/rails/example/mailbox_example_group_spec.rb

```diff
@@ -0,0 +1,83 @@
+require "spec_helper"
+require "rspec/rails/feature_check"
+
+class ApplicationMailbox
+  class << self
+    attr_accessor :received
+
+    def receive(*)
+      self.received += 1
+    end
+  end
+
+  self.received = 0
+end
+
+module RSpec
+  module Rails
+    describe MailboxExampleGroup, :skip => !RSpec::Rails::FeatureCheck.has_active_job? do
```

Comment:
this should check for `has_action_mailbox?` instead of `has_active_job?`..

Reply:
Thank you! Opened in https://github.com/rspec/rspec-rails/pull/2123